addons/website/static/src/js/content/menu.js

```diff
@@ -80,7 +80,7 @@ const BaseAnimatedHeader = animations.Animation.extend({
      * @private
      */
     _updateMainPaddingTop: function () {
-        this.headerHeight = this.$el.outerHeight();
+        this.headerHeight = this.$el.hasClass('o_header_standard') ? this.$el.outerHeight() : this.$el.find('.navbar').outerHeight();
```

Comment:
Does that `hasClass` condition have to be recomputed each time?

Reply:
not really